intg/src/main/java/org/apache/atlas/AtlasConfiguration.java

```diff
@@ -146,7 +146,18 @@ public enum AtlasConfiguration {
     ATLAS_BULK_API_MAX_ENTITIES_ALLOWED("atlas.bulk.api.max.entities.allowed", 10000),
 
     ENABLE_ASYNC_TYPE_UPDATE("atlas.types.update.async.enable", false),
-    MAX_THREADS_TYPE_UPDATE("atlas.types.update.thread.count", 3);
+    MAX_THREADS_TYPE_UPDATE("atlas.types.update.thread.count", 3),
+
+    // Classification propagation thread pool configuration
+    CLASSIFICATION_PROPAGATION_CORE_POOL_SIZE("atlas.classification.propagation.core.pool.size", 32),
+    CLASSIFICATION_PROPAGATION_MAX_POOL_SIZE("atlas.classification.propagation.max.pool.size", 200),
+    CLASSIFICATION_PROPAGATION_QUEUE_CAPACITY("atlas.classification.propagation.queue.capacity", 1000),
+    CLASSIFICATION_PROPAGATION_KEEP_ALIVE_SECONDS("atlas.classification.propagation.keep.alive.seconds", 300),
```

Comment:
I think these are for async notifications, please rename accordingly
E.g. TAG_ASYNC_LISTENER_* / TAG_ASYNC_NOTIFIER_*